frontend/src/screens/auth/Login/Login.module.css

```diff
@@ -0,0 +1,84 @@
+.loginContainer {
+  width: 100%;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.loginContent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 50vw;
+  max-width: 500px;
+  padding: 20px;
+  border-radius: 8px;
+  background-color: var(--primary);
+}
+
+.title {
+  margin: 0;
+  font-size: 3rem;
+  font-weight: 200;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 80%;
+}
+
+.inputGroup {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-bottom: 15px;
+}
+
+.label {
+  margin-bottom: 5px;
+  font-size: 15px;
+}
+
+.input {
+  padding: 8px;
+  border: none;
+  margin: 0;
+  border-radius: 4px;
+  font-size: 14px;
+}
+
+.buttonGroup {
```

Comment:
Not a blocker for now, but you can fix at the end. You probably want more paddings on the button group so that the padding among different components are the same for better visualization.

Reply:
Thanks for letting me know. I'll work on it in future PR's